spec/teacher_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './person'
+require './teacher'
+
+describe Teacher do
+  teacher = Teacher.new(30, 'Lucas', specialization: 'Math')
+
+  context 'should create instance of Teacher Class' do
+    it 'returns correct name' do
+      expect(teacher.name).to eq('Lucas')
+    end
+
+    it 'returns correct age' do
+      expect(teacher.age).to eq(30)
+    end
+
+    it 'should return correct data' do
+      expect(teacher.can_use_services?).to eq true
```

Comment:
- [ ] it would be better to add tests for `can_use_services` you should test positive case and negative case

Reply:
@youmari In the Teacher class `can_use_services?` will always return true, that was the project requirement. So there will be no negative case.